test/unit/setup/node.js

```diff
@@ -14,10 +14,10 @@ if (!global.document || !global.window) {
   var jsdom = require('jsdom').jsdom;
```

Comment:
Is this file in use?  It seems like you only want `test/setup/node.js` and not this one.